### PR TITLE
Bug 7290 Errors for declaration checkboxes not appearing

### PR DESCRIPTION
**Pull request by nickwhm**
Errors for declaration checkboxes not appearing have used the HMRCAppContext to identify the view by name, then I'm wrapping the checkboxes in their own individual groups. This means we get the red border and the error message is assigned correctly.

### CODE REVIEW

File: src/components/override-sdk/field/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -56,7 +59,27 @@ export default function CheckboxComponent(props) {
   return (
     <>    
       {exclusiveOption && <div className="govuk-checkboxes__divider">or</div>}
-      <GDSCheckbox
+
+      {/* If its the declaration view then group the checkboxes separately so the error message is assigned correctly */}
+      {OverrideLabelValue === 'Declaration' ? (
```
Comment (AkhilaHMRC) [inline src/components/override-sdk/field/Checkbox/Checkbox.tsx:64]: need to consider the localised label value as well for welsh

Comment (AkhilaHMRC) [reply src/components/override-sdk/field/Checkbox/Checkbox.tsx:64]: Make sure this code works for welsh as well

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]

File: src/components/override-sdk/field/Checkbox/Checkbox.tsx (hunk 1 above)
Comment (nickwhm) [reply src/components/override-sdk/field/Checkbox/Checkbox.tsx:64]: Screenshot 2023-11-17 131151

Comment (nickwhm) [reply src/components/override-sdk/field/Checkbox/Checkbox.tsx]: The functionality works for both English and Welsh. I've also added `trim()` and `toLowerCase()` just to make sure the text is always the same on the `OverrideLabelValue`.

**Review by AkhilaHMRC — APPROVED** [review]

**Pull request merged**